mobile_app_connector/models/mobile_feedback.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2019 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Jonathan Tarabbia
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+import logging
+
+from datetime import datetime
+from odoo import models, fields, _
+
+logger = logging.getLogger(__name__)
+
+
+class AppBanner(models.Model):
+    _name = 'mobile.app.feedback'
+    _description = 'Mobile App Feedback'
+    _order = 'id desc'
+
+    ##########################################################################
+    #                                 FIELDS                                 #
+    ##########################################################################
+    name = fields.Char('What did you like', translate=True, required=True,
+                       readonly=True)
+    partner_id = fields.Many2one('res.partner', 'Partner', readonly=True)
+    improve_app = fields.Char(translate=True, readonly=True)
+    source = fields.Char(translate=True, readonly=True)
```

Comment:
Should this be a Selection field (we control the possible source values), or is it something sent by the user that can be free text?